modules/vpc/outputs.tf

```diff
@@ -1,19 +1,29 @@
-output "vpc_id" {
-  value = module.vpc[0].vpc_id
+output "id" {
+  value       = try(module.vpc[0].vpc_id, null)
+  description = "The newly created vpc id"
 }
 
-output "vpc_private_subnets" {
-  value = module.vpc[0].private_subnets
+output "private_subnets" {
+  value       = try(module.vpc[0].private_subnets, null)
+  description = "The newly created vpc private subnets IDs list"
 }
 
-output "vpc_public_subnets" {
-  value = module.vpc[0].public_subnets
+output "public_subnets" {
+  value       = try(module.vpc[0].public_subnets, null)
+  description = "The newly created vpc public subnets IDs list"
 }
 
-output "vpc_cidr_block" {
-  value = module.vpc[0].vpc_cidr_block
+output "cidr_block" {
+  value       = try(module.vpc[0].vpc_cidr_block, null)
```

Comment:
Can't we use just `cidr_block` instead of `vpc_cidr_block` like other changed variables?

Reply:
this is a value coming from external module we use terraform-aws-modules/vpc/aws